docs/cs-proof.md

```diff
@@ -556,18 +556,18 @@ A_O &= \widetilde{B} \cdot \tilde{o} + \langle \mathbf{G} , \mathbf{a}\_O \rangl
 W_L &= \langle \mathbf{y}^{-n} \circ \mathbf{w}\_L, \mathbf{H} \rangle \\\\
 W_R &= \langle \mathbf{y}^{-n} \circ \mathbf{w}\_R, \mathbf{G} \rangle \\\\
 W_O &= \langle \mathbf{y}^{-n} \circ \mathbf{w}\_O, \mathbf{H} \rangle \\\\
-S &= \widetilde{B} \cdot \tilde{s} + \langle \mathbf{G} , \mathbf{s}\_L \rangle + \langle \mathbf{H}, \mathbf{s}\_R \rangle
+S   &= \widetilde{B} \cdot \tilde{s} + \langle \mathbf{G} , \mathbf{s}\_L \rangle + \langle \mathbf{H}, \mathbf{s}\_R \rangle
 \end{aligned}
 \\]
 
 The prover forms these commitments, and sends them to the verifier.
 
-For reference, here are the equations for \\({\mathbf{l}}(x)\\) and \\({\mathbf{r}}(x)\\):
+For reference, here are the equations for \\({\mathbf{l}}(x)\\) and transmuted \\({\mathbf{r}}(x)\\):
```

Comment:
In what sense is `r` transmuted?

Reply:
Ehh... i wanted to move `y^n` on the other side of the equation to match with the following 2d diagram. 

Maybe I should remove "transmuted" and add ", where the factor `y^n` is moved to the left side of the equation"

Reply:
Changed to 
> For reference, here are the equations for `l(x)`, and `r(x)` multiplied by `y^-n`

<img width="882" alt="image" src="https://user-images.githubusercontent.com/698/47120667-66ca7500-d224-11e8-972a-b7f87ddb2eca.png">